src/Microsoft.ApplicationInsights.AspNet/ApplicationInsightsExceptionMiddleware.cs

```diff
@@ -26,7 +27,9 @@ public ApplicationInsightsExceptionMiddleware(RequestDelegate next, TelemetryCli
             {
                 if (this.telemetryClient != null)
                 {
-                    this.telemetryClient.TrackException(exp);
+                    var exceptionTelemetry = new ExceptionTelemetry(exp);
+                    exceptionTelemetry.HandledAt = ExceptionHandledAt.Platform;
+                    this.telemetryClient.Track(exceptionTelemetry);
                 }
 
                 throw;
```

Comment:
Technically, the exception is unhandled and rethrown. Shouldn't we report it as HandledAt=Unhandled? What if there is no other middleware that handles exceptions?